xtask/codegen/css.ungram

```diff
@@ -417,8 +418,15 @@ CssDeclarationImportant =
 ////////////
 AnyCssAtRule =
 	CssAtKeyframes
+	| CssAtCharsetRule
 	| CssAtMedia
+	| CssBogusAtRule
 
+CssAtCharsetRule =
+	'@'
+	'charset'
+	encoding: CssString
+	';'
```

Comment:
Since we plan to have many at rules, what do you think about this variation?

```
CssAtRule =
	'@'
	rule: AnyCssAtRule

AnyCssAtRule = CssCharsetRule | CssKeyframeRule

CssCharsetRule = 
	'charset'
	encoding: CssString
	';'
```